queries.go

```diff
@@ -28,9 +28,11 @@ const (
 
 // Context constants
 const (
-	TIMEOUT          = "timeout"
-	SKIPEMPTYBUCKETS = "skipEmptyBuckets"
-	QUERYID          = "queryId"
+	TIMEOUT                  = "timeout"
+	SKIPEMPTYBUCKETS         = "skipEmptyBuckets"
+	QUERYID                  = "queryId"
+	POPULATECACHE            = "populateCache"
+        POPULATERESULTLEVELCACHE = "populateResultLevelCache"
```

Comment:
Weird white space issue, can you run go fmt first.

Reply:
Strange in the git repo and in my local repo it seems fine

Reply:
Must be the tag commit that fixed it. Weird it shows up here.

Reply:
I made another pull request that should address it